twisted/test/test_jelly.py

```diff
@@ -224,13 +232,18 @@ def test_typeBuiltin(self):
 
 
     def test_dateTime(self):
+        """
+        Jellying L{datetime.timedelta} instances and then unjellying the result
+        should produce objects which represent the values of the original
+        inputs.
+        """
         dtn = datetime.datetime.now()
         dtd = datetime.datetime.now() - dtn
-        input = [dtn, dtd]
-        c = jelly.jelly(input)
+        inputList = [dtn, dtd]
```

Comment:
I used inputList to keep it up with the variable naming style of other tests. For example : https://github.com/twisted/twisted/blob/trunk/twisted/test/test_jelly.py#L242, https://github.com/twisted/twisted/blob/trunk/twisted/test/test_jelly.py#L282,
https://github.com/twisted/twisted/blob/trunk/twisted/test/test_jelly.py#L304.
Besides input is also a keyword in Python3 not that it causes any problem.